distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -95,10 +101,24 @@ grant
     permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
 };
 grant
+  codeBase "file:${ddf.home.policy}lib/ext/-" {
```

Comment:
❓ Isn't this granting a lot to way more than we want? If anything in lib/ext is allowed to set policy, insert or remove providers ... aren't we just opening it up to something which could change it all?

Reply:
It's more open than I would prefer. It's as open as I believe we need it.